lib/new_relixir/agent.ex

```diff
@@ -138,7 +138,6 @@ defmodule NewRelixir.Agent do
     Enum.join(segments)
   end
 
-  defp urljoin([]), do: []
```

Comment:
Dialyzer checks that this function is called over a list that is never empty. As you can see [here](https://github.com/TheRealReal/new-relixir/blob/9a0635fa337b355a249bd0f337497a291339a117/lib/new_relixir/agent.ex#L137) we call it with 
```elixir
args ++ base_args
```
which results in a non-empty list given the value of [`base_args`](https://github.com/TheRealReal/new-relixir/blob/9a0635fa337b355a249bd0f337497a291339a117/lib/new_relixir/agent.ex#L131)

Reply:
👍 

Do you mind changing `base_args` to a module attr?

Reply:
Thanks for your review @sudostack - I just pushed the update to change `base_args` to make it a module attr. But I had to keep `license_key: license_key()` out of it given that we cannot call the function on the module attr definition